### PR TITLE
Adding migration status and cutover to extension 

### PR DESCRIPTION
**Pull request by aasimkhan30**
New Dashboard UX
image

Now you can see count of selected databases
image

Migration Status Table
image

Migration Cutover dialog
image

Migration completed
image

### CODE REVIEW

File: extensions/sql-migration/src/api/azure.ts [hunk 1]
```diff
@@ -171,17 +173,48 @@ export async function startDatabaseMigration(account: azdata.Account, subscripti
...
+	const response = await api.makeAzureRestRequest(account, subscription, path, azurecore.HttpRequestMethod.GET, undefined, true, host);
+	if (response.errors.length > 0) {
+		throw new Error(response.errors.toString());
+	}
+	return response.response.data.value;
+}
+
+export async function startMigrationCutover(account: azdata.Account, subscription: Subscription, migrationStatus: DatabaseMigration): Promise<any> {
+	const api = await getAzureCoreAPI();
+	const host = `https://eastus2euap.management.azure.com`;
```
Comment (kburtram) [inline extensions/sql-migration/src/api/azure.ts:211]: should the host come from a config file or something?

File: extensions/sql-migration/src/api/azure.ts [hunk 2]
```diff
@@ -285,19 +318,105 @@ export interface StartDatabaseMigrationRequest {
...
+	activeBackupSets: BackupSetInfo[];
+	blobContainerName: string;
+	isFullBackupRestored: boolean;
+	restoreBlockingReason: string;
+	fileUploadBlockingErrors: string[];
+	currentRestoringFileName: string;
+	lastRestoredFilename: string;
+}
+
+export interface SqlConnectionInfo {
```
Comment (kburtram) [inline extensions/sql-migration/src/api/azure.ts:361]: do we really need new connection type?  i.e. we can use on of the existing ones?

**Review by kburtram — APPROVED** [review]
LGTM as WIP

**Comment by alanrenmsft** [comment]
screenshots for dark theme and High Contrast?

File: extensions/sql-migration/src/api/azure.ts (hunk 2 above)
Comment (aasimkhan30) [reply extensions/sql-migration/src/api/azure.ts:361]: This is the connection Info object from api definitions. It is only used in azure stuff.

**Comment by aasimkhan30** [comment]
image
image
image


@alanrenmsft there is some work needed to make it compatible with dark mode.

File: extensions/sql-migration/src/api/azure.ts (hunk 2 above)
Comment (kburtram) [reply extensions/sql-migration/src/api/azure.ts:361]: why does it need to be redeclared here?

**Pull request merged**